src/GoogleFitOnFhir.UnitTests/RoutingServiceTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using GoogleFitOnFhir.Common;
+using GoogleFitOnFhir.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Health.Common.Handler;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using Xunit;
+using ExecutionContext = Microsoft.Azure.WebJobs.ExecutionContext;
+
+namespace GoogleFitOnFhir.UnitTests
+{
+    public class RoutingServiceTests
+    {
+        private readonly PathString _anyRequest = "/api/";
+        private readonly PathString _validRequest = "/api/manufacturer/operation";
+
+        private readonly ILogger<RoutingService> _logger;
+        private readonly IResponsibilityHandler<RoutingRequest, Task<IActionResult>> _handler;
+        private readonly IRoutingService _routingService;
+
+        // private static string _fakeRedirectUri = "http://localhost";
```

Comment:
Let's delete this.

Reply:
Thought I was going to use this and didn't, but forget to clean it up.